### DISCUSSION TITLE
Item Interaction

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#839] [Help] [answered]
Hello! I have the event `PlayerBlockInteractEvent.class` registered but it only works when clicking on a block not with the air. And I also only found `PlayerEntityInteractEvent.class`. But is there an Event that represents Bukkits `PlayerInteractEvent` so it also works when rightclicking air?

Thanks :D

**Comment by CraziiDust** [comment, marked as answer]
This event fired when use item, but doesn't work when the player doesn't hold any item
[PlayerUseItemEvent](https://javadoc.minestom.net/net/minestom/server/event/player/PlayerUseItemEvent.html)

**Reply by DasLixou** [reply]
It helps, thanks, but when i hold the item and for example open a trapdoor, it doenst get called.

**Reply by emortaldev** [reply]
Then you would use PlayerUseItemOnBlockEvent

**Reply by DasLixou** [reply]
thanks @emortaldev 👍

**Reply by DasLixou** [reply]
but is there a way to make multiple events in one listener @emortaldev ?
